src/transactions/feed/detailContent/FeeRowItem.tsx

```diff
@@ -30,26 +38,29 @@ export default function FeeRowItem({ fees, feeType, transactionStatus }: Props)
   }
 
   return (
-    <View testID="TransactionDetails/FeeRowItem">
-      <View style={styles.row}>
-        <Text style={styles.bodyText}>{label}</Text>
+    <View style={styles.row} testID="TransactionDetails/FeeRowItem">
```

Comment:
moving the Views around so that long labels can wrap to 2 lines without affecting the right hand values